apps/cowswap-frontend/src/legacy/state/swap/TradeGp.ts

```diff
@@ -117,29 +120,37 @@ export default class TradeGp {
    * The partner fee
    */
   readonly partnerFee?: PartnerFee
+  /**
+   * The partner fee as token amount
+   */
+  readonly partnerFeeAmount?: CurrencyAmount<Currency>
```

Comment:
Will we save this in sell token, or will depend if its a buy order of sell order? 
I guess always sell order, but want to check. If so, should we reflect it in the name or description?